acceptancetests/test_existing.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
I don't think we need a PoC to land in the tree, but this is useful for vetting. This should just be a straight make assess though right?